x-pack/plugins/apm/public/hooks/use_transaction_charts_fetcher.ts

```diff
@@ -9,17 +9,19 @@ import { useParams } from 'react-router-dom';
 import { getTransactionCharts } from '../selectors/chart_selectors';
 import { useFetcher } from './use_fetcher';
 import { useUrlParams } from '../context/url_params_context/use_url_params';
+import { useApmServiceContext } from '../context/apm_service/use_apm_service_context';
 
 export function useTransactionChartsFetcher() {
   const { serviceName } = useParams<{ serviceName?: string }>();
+  const { transactionType } = useApmServiceContext();
   const {
-    urlParams: { transactionType, start, end, transactionName },
+    urlParams: { start, end, transactionName },
     uiFilters,
   } = useUrlParams();
 
   const { data, error, status } = useFetcher(
     (callApmApi) => {
-      if (serviceName && start && end) {
+      if (serviceName && start && end && transactionType) {
         return callApmApi({
           endpoint:
             'GET /api/apm/services/{serviceName}/transaction_groups/charts',
```

Comment:
Can you make `transactionType` required on the following endpoints:

 - GET /api/apm/services/{serviceName}/transaction_groups/charts
 - GET /api/apm/services/{serviceName}/transaction_groups/error_rate